src/sentry/static/sentry/app/components/organizations/timeRangeSelector/dateSummary.tsx

```diff
@@ -10,39 +9,26 @@ import {DEFAULT_DAY_END_TIME, DEFAULT_DAY_START_TIME} from 'app/utils/dates';
 import {t} from 'app/locale';
 import space from 'app/styles/space';
 
-class DateSummary extends React.Component {
-  static propTypes = {
-    /**
-     * Start date value for absolute date selector
-     * Accepts a JS Date or a moment object
-     *
-     * React does not support `instanceOf` with null values
-     */
-    start: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+type Props = {
+  start: Date;
+  end: Date;
+};
 
-    /**
-     * End date value for absolute date selector
-     * Accepts a JS Date or a moment object
-     *
-     * React does not support `instanceOf` with null values
-     */
-    end: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
-  };
-
-  getFormattedDate(date, format) {
+class DateSummary extends React.Component<Props> {
+  getFormattedDate(date: Date, format: string) {
     return moment(date).local().format(format);
   }
 
-  formatDate(date) {
+  formatDate(date: Date) {
     return this.getFormattedDate(date, 'll');
   }
 
-  formatTime(date, withSeconds = false) {
+  formatTime(date: Date, withSeconds = false) {
     return this.getFormattedDate(date, `HH:mm${withSeconds ? ':ss' : ''}`);
   }
 
   render() {
-    const {className, start, end} = this.props;
+    const {start, end} = this.props;
```

Comment:
wasn't `className` being used?

Reply:
Hmmm from what I can see DateSummary is only used in one place and the only props it receives are start and end, but I could be missing something.